src/test/java/showcase/neodymium/tests/pdf/PdfTextContentCheckTest.java

```diff
@@ -23,11 +23,8 @@
  * implemented in Java and the execution environment is configured with Maven any available 3rd party technology can be
  * added to the test automation.<br>
  * This show case gives an basic example how to implement an PDF content check using pdfbox.<br>
- * <br>
- * <b>REQUIRED CONFIGURATION</b> <i>pom.xml</i>:
- * <ul>
- * <li>org.apache.pdfbox/pdfbox must be added to the dependencies section</li>
- * </ul>
+ * In order to be able to use pdfbox, a dependency entry <i>org.apache.pdfbox</i> is required in the <i>pom.xml but
```

Comment:
In order to use PDFbox you would need to add a dependency for <i>org.apache.pdfbox</i> to your project's <i>pom.xml</i> file. For this show case project we already prepared this and you can have a look if you're struggling with the setup.<br>